src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildTrigger.java

```diff
@@ -284,7 +284,12 @@ private boolean hasCauseFromTheSamePullRequest(@Nullable List<Cause> causes, @Nu
         ParametersDefinitionProperty definitionProperty = this.job.getProperty(ParametersDefinitionProperty.class);
         if (definitionProperty != null) {
             for (ParameterDefinition definition : definitionProperty.getParameterDefinitions()) {
-                values.add(definition.getDefaultParameterValue());
+                ParameterValue defaultValue = definition.getDefaultParameterValue();
+                if (defaultValue == null) {
+                    logger.warning(format("No default value for the parameter '%s'.", definition.getName()));
```

Comment:
I don't think this should be a warning. With frequent-running jobs it can spam the log.

It would be much better to output this to build log. If that's too difficult here I would just reduce the log level to debug/fine.

Reply:
To store data before the build starts, we need a custom Action. PR #43 removes `StashPostBuildCommentAction`, I guess it can be repurposed to hold warnings from the trigger stage, which can later be printed to the build log and posted to Stash.
Let me do `logger.fine` for now, it's better than crashing.